scripts/release.sh

```diff
@@ -122,6 +154,51 @@ function parse_flags() {
     shift
   done
 
+  # Setup dot releases
+  if (( is_dot_release )); then
+    echo "Dot release requested"
+    TAG_RELEASE=1
+    PUBLISH_RELEASE=1
+    # List latest release
+    local releases
```

Comment:
[nit] can be combined local releases="$(hub tool release)"

Reply:
No, it can't. If you do that the successful attribution will override the subshell return code, and you'll never catch errors from hub.